usr/sbin/registercloudguest

```diff
@@ -88,8 +88,8 @@ def register_modules(extensions, products, registered=[]):
                 stderr=subprocess.PIPE
             )
             res = p.communicate()
-            if p.returncode != 0:
-                registration_returncode = p.returncode
+            registration_failed = p.returncode != 0
```

Comment:
Nitpick: this is a bit difficult to read. Could make it clearer with a "ternary conditional".

```
registration_failed = False if p.returncode == 0 else True
```

Or maybe it could be simplified to:

```
registration_failed = True if p.returncode else False
```

If we know return code is always an integer and non zero values are all failures.

Reply:
While I addressed the suggestion, I wonder if it would be simpler 
```python
registration_failed = p.returncode
```
and have 0 and > 0 instead of False and True